tests/manual_download_data.html

```diff
@@ -184,12 +184,12 @@
         return Module['downloadedData'];
       }
 
-      var dataDownload = download('manual_download_data.data').then(function(data) {
+      var dataDownload = download('/test/manual_download_data.data').then(function(data) {
         console.log('downloaded data file');
-        Module.manuallyDownloadedData = 1;
         Module['downloadedData'] = data;
         var jsDownload = download('manual_download_data.js').then(function(data) {
           console.log('downloaded js file');
+          Module['mainScriptUrlOrBlob'] = new Blob([data]);
```

Comment:
What does this change do?

Reply:
It is used by the worker to load the glue file - https://github.com/emscripten-core/emscripten/blob/df034c938c8bdbb3b0a272e9794084e9bce8567c/src/library_pthread.js#L330

Reply:
> It is used by the worker to load the glue file -
> 
> https://github.com/emscripten-core/emscripten/blob/df034c938c8bdbb3b0a272e9794084e9bce8567c/src/library_pthread.js#L330

So it wasn't needed before because this test didn't run with workers/threads?

Reply:
If you don't do this will the test not pass?

Reply:
>  So it wasn't needed before because this test didn't run with workers/threads?
yes

> If you don't do this will the test not pass?

yes, pthread test won't pass